apps/playground/src/app/components/ag-grid/data-manager-large/data-manager-large.component.ts

```diff
@@ -72,6 +72,9 @@ export class DataManagerLargeComponent implements OnInit {
             cellRendererFramework: CustomLinkComponent,
           },
         },
+        context: {
+          enableTopHorizontalScroll: true,
```

Comment:
Change to `enableTopScroll` since "horizontal" is redundant (we probably won't have a top vertical scrollbar, for instance).

Reply:
Renamed.